docs/_docs/Instructors/02-Teachers-How-to-build-Learning-Units.md

```diff
@@ -173,48 +291,39 @@ development will be done on top of the previous batch.
 ## Instructors repository 
 
 As mentioned in the [Teaching Process](./01-Teaching-process.md), the Learning Units 
-are pushed to the [Academy GitHub](https://github.com/LDSSA/), 
-to the repository with the name format `batchX-instructors`.
-
+are pushed to the [Academy GitHub](https://github.com/LDSSA/), to the repository with the 
+name format `batchX-instructors`. The README in the repository contains detailed 
+instructions around the batch process's technical aspects, such as branch naming, requirement,
+etc.
 
 Once the base improvements are done, the unit is shared with the QA team, which will 
 review and test the unit. This feedback helps us detect potential bugs and places for 
 improvement before the materials reach the students.
 
-### Adding Units
+Recommended process whe adding units:
 
-> ⚠️ Section in Progress 
-
-Suggested execution order:
-1. Carefully read the scope of your learning unit in the Curriculum repo, and think about what the students should be able to do after they have finished  
-2. Go through the learning notebooks and check for anything that needs improvements
-3. Google around and see how other people teach the concepts. 
-4. Submit the Learning Notebook and Example Notebook's improvements 
-5. [Optional] Share with QA  
+1. Open an issue and an associated PR to indicate the unit is being tracked and developed
+2. Carefully read the scope of your learning unit in the Curriculum repo, and think about what the students should be able to do after they have finished  
```

Comment:
Add a link to https://github.com/LDSSA/curriculum-development here? (although it needs some updating...)

Reply:
Added 👍